BridgeEmulator/protocols/mqtt.py

```diff
@@ -1,13 +1,11 @@
 import json
 import logging
+import random
 import paho.mqtt.client as mqtt
 
 from functions import light_types, nextFreeId
 from functions.colors import hsv_to_rgb
 
-# Mqtt client creation
-# You will need to keep this around, because it will manage all the pushed messages
-client = mqtt.Client()

```

Comment:
You cannot remove the creation of the client.

Reply:
I test this and with client creation under mqttServer function the bridge_config is passed to handlers functions. Not really sure where is the perfect place for client creation, uder the function or outside. I put in under the function because we execute the function from HueEmulator3.py file, not the file. Is up to you to decide where is the best place, important is to work.